src/environments/environment.ts

```diff
@@ -14,21 +14,25 @@ export const environment = {
   fineractPlatformTenantId: window['env']['fineractPlatformTenantId'] || 'default',
   // For connecting to others servers running elsewhere update the base API URL
   baseApiUrls: window['env']['fineractApiUrls'] ||
-    'https://dev.mifos.io,https://demo.mifos.io,https://qa.mifos.io,https://staging.mifos.io,https://mobile.mifos.io,https://demo.fineract.dev,https://localhost:8443',
+    'https://dev.mifos.io,https://demo.mifos.io,https://qa.mifos.io,https://staging.mifos.io,https://mobile.mifos.io,https://loans.qa.oneacrefund.org,https://localhost:8443',
   // For connecting to server running elsewhere set the base API URL
-  baseApiUrl: window['env']['baseApiUrl'] || 'https://demo.fineract.dev',
+  baseApiUrl: window['env']['baseApiUrl'] || 'https://loans.qa.oneacrefund.org',
   allowServerSwitch: env.allow_switching_backend_instance,
   apiProvider: window['env']['apiProvider'] || '/fineract-provider/api',
   apiVersion: window['env']['apiVersion'] || '/v1',
   serverUrl: '',
   oauth: {
-    enabled: false,  // For connecting to Mifos X using OAuth2 Authentication change the value to true
-    serverUrl: ''
+    enabled: true,  // For connecting to Mifos X using OAuth2 Authentication change the value to true
+    serverUrl: 'https://accounts.qa.oneacrefund.org/auth',
+    realm:'OneAcreFund',
+    client_id:'fineract',
+    redirectUri:'http://localhost:4200/home'
```

Comment:
I am not sure if this URL will be reachable, Will it be in the QA env?

Reply:
for now the tests used are the QA environment, he has issues on the keycloak test.